manifests/cli.pp

```diff
@@ -93,7 +93,11 @@
         'foreman/hammer_root.yml.epp',
         {
           username => $username_real,
-          password => $password_real,
+          # epp() returns a Content of Datatype Sensitive, if at least one Parameter is of Datatype Sensitive.
+          # The "unwrap" therefore is not necessary, BUT: I do not know, how to make the Test pass in
+          # spec/classes/foreman_cli_spec.rb:110 otherwise.
```

Comment:
Perhaps https://github.com/theforeman/puppet-foreman/pull/976 would help with that.

Reply:
Can this now be resolved?

Reply:
I removed the Comment and the `unwrap`, but did not manage to establish a Rspec-Test.  
I amended the Commit.